tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -183,27 +132,17 @@
         - name: ANSIBLE_FORCE_COLOR
           value: "True"
       nodes:
-        standalone:
-          hostName: standalone
+        {{ edpm_node_hostname }}:
+          hostName: {{ edpm_node_hostname }}
           ansible:
             ansibleHost: {{ edpm_node_ip }}
-          networks:
-          - defaultRoute: true
-            fixedIP: {{ edpm_node_ip }}
-            name: CtlPlane
-            subnetName: subnet1
-          - name: InternalApi
-            subnetName: subnet1
-          - name: Storage
-            subnetName: subnet1
-          - name: Tenant
-            subnetName: subnet1
+          networks: {{ edpm_node_networks }}
       nodeTemplate:
         ansibleSSHPrivateKeySecret: {{ ansible_ssh_private_key_secret }}
         managementNetwork: ctlplane
         ansible:
           ansiblePort: 22
-          ansibleUser: root
+          ansibleUser: {{ edpm_user }}
           ansibleVars:
             os_net_config_iface: {{ dataplane_os_net_config_iface | default ('nic1') }}
```

Comment:
missing             os_net_config_dns: {{ dataplane_os_net_config_dns | default("") }}

Reply:
it isn't missing, it isn't captured in the diff because it is already there, if you expand the line below here you will see it

Reply:
oh I am sorry my bad 

ok, i am also running the test now with this at https://review.rdoproject.org/r/c/testproject/+/50653/14//COMMIT_MSG#11
https://review.rdoproject.org/zuul/status#50653 if that is green then lets merge this 

we'll be wiring it up today with https://review.rdoproject.org/r/c/config/+/50964 so it should run on pull requests from this point on